spec/integration/query_cache_spec.rb

```diff
@@ -1030,15 +1014,47 @@
       authorized_collection.find.to_a
     end
 
-    let(:events) do
-      subscriber.command_started_events('find')
-    end
-
     it 'queries again' do
       new_collection.find.to_a
       expect(Mongo::QueryCache.send(:cache_table).length).to eq(2)
       expect(events.length).to eq(2)
     end
   end
 
+  context 'with system collection' do
+    let(:client) do
+      ClientRegistry.instance.global_client('root_authorized').tap do |client|
+        client.subscribe(Mongo::Monitoring::COMMAND, subscriber)
+      end
+    end
```

Comment:
@p-mongo is it okay that I'm using the root authorized client here?

Reply:
Yes this makes sense to me, our authorized user I believe is scoped to the default test database hence to operate on users the root authorized user appears appropriate.